### PR TITLE
docs(datepicker): add demo for Show/Hide/Toggle events

### PR DESCRIPTION
**Pull request by ghost**
docs(datepicker): add demo for Show/Hide/Toggle datepicker's events
related to #3340

### CODE REVIEW

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { DemoDatepickerMinMaxComponent } from './demos/min-max/min-max.component
 import { DemoDatepickerDisabledComponent } from './demos/disabled/disabled.component';
 import { DemoDatepickerFormsComponent } from './demos/forms/forms.component';
 import { DemoDatepickerReactiveFormsComponent } from './demos/reactive-forms/reactive-forms.component';
+import { DemoDatepickerEventsComponent } from './demos/events/events.component';
```
Comment (EvilAlexei) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: Not events, this is open-hide methods. Folder, filenames and component name

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

**Review by EvilAlexei — CHANGES_REQUESTED** [review]
It's not just methods, they are very specific methods :)
visibility-methods or presence-methods, something like that.

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 2]
```diff
@@ -131,6 +132,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/reactive-forms/reactive-forms.component.ts'),
         html: require('!!raw-loader?lang=markup!./demos/reactive-forms/reactive-forms.component.html'),
         outlet: DemoDatepickerReactiveFormsComponent
+      },
+      {
+        title: 'Visibility methods',
+        anchor: 'visibility-methods',
+        component: require('!!raw-loader?lang=typescript!./demos/visibility-methods/visibility-methods.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/visibility-methods/visibility-methods.html'),
+        description: `<p>Was used <code>outsideClick</code> property to show how to work methods</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: change description to : You can manage datepicker's state by using its `show()`, `hide()` and `toggle()` methods

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Review by EvilAlexei — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Will be better if show(), hide(). toggle() methods in description were marked as code (using html)
And datepicker popup should be closed by missclick (like other datepicker examples on demo page)
smallissue2
Otherwise, looks good

**Comment by EvilAlexei** [comment]
In this demo datepicker shouldn't close on `outsideClick`. There are two methods that can close it, hide and toggle. If yoy remove outsideClick -> button `open` will close datepicker too and this will be strange.

But i think we should add a not for `outsideClick` and why we are using it here

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Please, update branch from latest development and fix failed e2e tests, if it still be neccessary

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**